addons/website_slides/controllers/main.py

```diff
@@ -138,6 +138,10 @@ def _get_slide_quiz_data(self, slide):
                 } for answer in question.sudo().answer_ids],
             } for question in slide.question_ids]
         }
+        if 'slide_answer_quiz' in request.session:
+            slide_answer_quiz = json.loads(request.session['slide_answer_quiz'])
+            if str(slide.id) in slide_answer_quiz:
+                values['answers'] = slide_answer_quiz[str(slide.id)]
```

Comment:
It took me about 10 minuts to redo the flow and find where this 'answers'  variable came ... Could probably be renamed because we have a lot of "answers" :D . session_quiz_answers ?